hack/kubernetes_version/update_kubernetes_version.go

```diff
@@ -53,17 +102,19 @@ func main() {
 	mode := info.Mode()
 
```

Comment:
Do you mind adding a logging line describing the value of `vDefault` and `vNewest` that was discovered? It's difficult to tell if this is actually doing anything with the current repository state.

Reply:
yes, it makes a lot of sense - i'll add a logging line describing the value of `vDefault` and `vNewest` that was discovered

Reply:
add in new pr